src/constants/auth/auth.js

```diff
@@ -0,0 +1,12 @@
+export const DUMMY_LOGINS = [
```

Comment:
In this case, it should be in `src/mocks/auth.js` or `/credentials.js`. Those are `DUMMY` or to be certain `MOCK` data files